server/helios/utils.py

```diff
@@ -207,3 +215,17 @@ def lock_row(model, pk):
         pass
 
     return row
+
+
+def create_qr_code(data) -> PIL.Image:
```

Comment:
If you add data for return value, maybe add also for `data` param.

Reply:
We still don't know the format of the data